src/Foreign/Lua/Module/System.hs

```diff
@@ -75,6 +88,16 @@ pushModule = do
   Lua.addfunction "with_env" with_env
   Lua.addfunction "with_tmpdir" with_tmpdir
   Lua.addfunction "with_wd" with_wd
+  Lua.addfunction "take_directory"    take_directory
```

Comment:
Maybe add a visual aid indicating that filepath operations follow below. Note that we order functions alphabetically to make them easy to find.